packages/core/package.json

```diff
@@ -4,7 +4,15 @@
   "description": "",
   "main": "dist/umd/index.js",
   "types": "dist/esm/index.d.ts",
-  "module": "dist/esm/index.js",
+  "module": "src/index.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/umd/index.js",
```

Comment:
- try to check if node 16 plus is there
- try to see if require and import be the same to be esm to check the failed import
- try to publish cjs again and put it in the require as the last resort